iroh-store/src/store.rs

```diff
@@ -137,6 +137,7 @@ impl Store {
                 let mut iter = db.full_iterator_cf(&cf_meta, IteratorMode::End);
                 let last_id = iter
                     .next()
+                    .and_then(|r| Some(r.unwrap()))
```

Comment:
Not happy with this unwrap here. Any ideas for a better treatment?

Reply:
r.ok()